src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -773,6 +788,150 @@ tests:
               - name: "SceneID"
                 value: 0x01
 
+    - label:
+          "Step 8f: TH sends a AddScene command to DUT with the GroupID field
+          set to G1, the SceneID field set to 0x01, the TransitionTime field set
+          to 1000 (1s) and extension field sets holding an invalid
+          ExtensionField (Invalid attribute ID for given cluster). This should
+          fail and return a status of 0x85 (INVALID_COMMAND)."
+      PICS: S.S.C00.Rsp && PICS_SDK_CI_ONLY
```

Comment:
Why does this not work for real devices? Is this because the user needs to send in a scenable attribute ID that doesn't exist? For yaml, you should be able to pass this in as a config variable. If you opted to do this in python, you could just find one by looking through the attribute lists for the various endpoints.

Same for the other mandatory commands below.

Reply:
Because for this command to pass, the OnOff cluster must be supported, otherwise the Scenes cluster would just skip the cluster and store the scene, thus not generating an error.

So this AddScene command must hold the extension field set of a cluster supported on the DUT. 

Same for the other commands

Reply:
Currently the clusters with scenes attributes are:
ColorControl
DoorLock
LevelControl
ModeBase
ModeSelect
OnOff
Thermostat
WindowCovering

Having a scenes cluster without any of those would be okay, meaning the device could still store scenes with their names transition time, groupid and sceneid. their extensionfield sets would be empty, which would mean the scenes would not take any actions when being recalled